index.html

```diff
@@ -0,0 +1,581 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio- Bhaskar Ranjan Bora</title>
+    <!-- Using BootStrap CSS via CDN -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+  </head>
+  <body>
+    <main data-bs-spy="scroll" data-bs-target="#navbar-example2">
+      <header>
+        <nav id="navbar-example2" class="navbar navbar-expand-lg px-5 fixed-top bg-white">
+          <div class="container-fluid">
+            <a class="navbar-brand" href="#">My Logo</a>
```

Comment:
anchor tags with an invalid link will never solve the purpose of it. I would suggest using valid links inside \<a\> tags